paasta_tools/chronos_tools.py

```diff
@@ -478,10 +479,8 @@ def create_complete_config(service, job_name, soa_dir=DEFAULT_SOA_DIR):
     complete_config['description'] = get_config_hash(complete_config)
 
     # If the job was previously stopped, we should stop the new job as well
-    # NOTE this clobbers the 'disabled' param specified in the config file!
-    if desired_state == 'start':
-        complete_config['disabled'] = False
-    elif desired_state == 'stop':
+    # NOTE: this clobbers the 'disabled' param specified in the config file!
+    if desired_state == 'stop':
```

Comment:
I don't think we have to do this anymore.

What if we had the `get_disabled` for chronos jobs be just like `get_instances` in marathon? (where get_instances returns 0 when something is stopped)